gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -541,6 +788,40 @@ public static OperationFilter notEquals(OperationField field, long value) {
   }
 
   /**
+   * Class for filtering address lists.
+   */
+  class AddressFilter extends ListFilter {
+
+    private static final long serialVersionUID = -227481644259653765L;
+
+    AddressFilter(AddressField field, ComparisonOperator operator, Object value) {
+      super(field.selector(), operator, value);
+    }
+
+    /**
+     * Returns an equality filter for the given field and string value. For string fields,
+     * {@code value} is interpreted as a regular expression using RE2 syntax. {@code value} must
+     * match the entire field.
+     *
+     * @see <a href="https://github.com/google/re2">RE2</a>
```

Comment:
Globally, we could replace the RE2 links with the link you used [here](https://github.com/GoogleCloudPlatform/gcloud-java/pull/728/files#diff-fde05ccfd12fc9c19f55f238ee1bc75dR347) in your previous PR.